src/layouts/hass-subpage.ts

```diff
@@ -119,7 +119,6 @@ class HassSubpage extends LitElement {
           font-size: 20px;
           height: var(--header-height);
           padding: 8px 12px;
-          pointer-events: none;
```

Comment:
Why did you have this?

Reply:
No idea, looked it up in history and has been there since the first commit...